spec.bs

```diff
@@ -184,6 +192,43 @@ Issue(44): Consider accepting an array of contributions.
 
 </div>
 
+<div algorithm>
+The <dfn method for="PrivateAggregation">contributeToHistogramOnEvent(|event|,
+|contribution|)</dfn> method steps are:
+
+1. Let |defaultProcessingResult| be the result of running [=this=]'s
+    [=PrivateAggregation/should perform default contributeToHistogramOnEvent()
+    processing=] given |event| and |contribution|.
```

Comment:
Maybe we should move the `event` parsing steps to the top, and only call "should perform default contributeToHistogramOnEvent() processing" when we're sure it's not an internal event?

Reply:
For Protected Audience, we'll need to override all calls (due to the different types and the [special IG handling](https://github.com/patcg-individual-drafts/private-aggregation-api/blob/main/error_reporting.md#associating-error-events-with-a-single-interest-group)); so I don't think we can do this.

(I'm also doing my best to have this behave identically to just defining the IDLs separately so we can do that in the underlying implementation for simplicity.)